sql-schema/223.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE `sensors` MODIFY `sensor_divisor` BIGINT(20) NOT NULL DEFAULT '1';
```

Comment:
we have the potential for another schema change for sensors here. I'd like to get that into this PR so we only do one overall change to the table. Just waiting on confirmation that it fixes another issue.